packages/calypso-e2e/src/lib/pages/signup/signup-pick-plan-page.ts

```diff
@@ -45,7 +45,7 @@ export class SignupPickPlanPage {
 		}
 
 		const actions = [
-			this.page.waitForResponse( /.*sites\/new\?.*/ ),
+			this.page.waitForResponse( /.*sites\/new\?.*/, { timeout: 30 * 1000 } ),
```

Comment:
I got hit by a slow site creation locally. If we're willing to wait 30s for the page URL (below) then we should be willing to wait for site creation too